Dynatrace/dynatrace-operator
# Add connectionInfo to ProcessModuleConfig

# Description
Provisioner's reconciler periodically fetches connection info from the tenant and update the `ruxitagentproc.conf` file in csi driver storage.

## How can this be tested?
1. apply dynakube with `hostMonitoring` or `cloudNativeFullStack` enabled
2. check if `cat /data/<TENANT>/bin/<AGENT_VERSION>/agent/conf/ruxitagentproc.conf` contains correct (provided by tenant) fields:
```
[general]                         
tenant <TOKEN>
tenantToken <TOKEN>
...
serverAddress {https://<URL>/communication,...}
```

## Checklist
